command/agent/remote_exec_test.go

```diff
@@ -10,8 +10,17 @@ import (
 
 	"github.com/hashicorp/consul/consul/structs"
 	"github.com/hashicorp/consul/testutil"
+	"github.com/hashicorp/go-uuid"
 )
 
+func generateUUID() (ret string) {
+	var err error
+	if ret, err = uuid.GenerateUUID(); err != nil {
+		return "DEADC0DE-BADD-CAFE-D00D-FEEDFACECAFE"
```

Comment:
These might be better done as a `panic`, since this could cause weirdness in the tests if we hit this.